hardhat.config.ts

```diff
@@ -0,0 +1,97 @@
+import "@nomicfoundation/hardhat-chai-matchers";
+import "@nomiclabs/hardhat-ethers";
+import { ethers } from "ethers";
+import fs from "fs";
+import { HardhatUserConfig, task } from "hardhat/config";
+
+require("dotenv").config();
+const DEPLOYER_PRIVATE_KEY = process.env.DEPLOYER_PRIVATE_KEY;
```

Comment:
update `DEPLOYER_PRIVATE_KEY` in `env.example`

Reply:
would prefer to update `env.example` for all the process.env variables.

Reply:
fixed